changelog.md

```diff
@@ -8,6 +8,7 @@
 * Added functionality to split tracks in the Kymotracking widget.
 * Added functionality to provide minimum and maximum observation times per data point to `DwelltimeModel` and `DwelltimeBootstrap`.
 * Enabled adding `KymoTrackGroup`s with tracks from different source kymographs. *Note that certain features (e.g., `KymoTrackGroup.ensemble_msd()` and `KymoTrackGroup.ensemble_diffusion("ols")`) require that all source kymographs have the same pixel size and scan line times.*
+* Added support for dwell time analysis for `KymoTrackGroup` with multiple source `Kymo`s.
```

Comment:
I wonder if we actually need this? Considering the feature to allow for multiple source kymographs is introduced in the same version, this *just works* rather than being a special case, no?

Reply:
Probably no need indeed.